summertime/dataset/dataset_loaders.py

```diff
@@ -28,6 +28,7 @@ class CnndmDataset(SummDataset):
     is_query_based = False
     is_dialogue_based = False
     is_multi_document = False
+    is_multilingual = False
```

Comment:
Oh, no, you don't have to  specify them in every class, you only need to define it once in the parent class. If it's not overridden in the child class, it will stay unchanged. So you can probably move this (`is_multilingual = False`) to the `SummDataset` instead?